flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/CliClient.java

```diff
@@ -702,14 +698,14 @@ private void closeTerminal() {
         }
     }
 
-    private LineReader createLineReader(Terminal terminal) {
+    private LineReader createLineReader(Terminal terminal, Completer completer) {
```

Comment:
```suggestion
    private LineReader createLineReader(Terminal terminal, @Nullable Completer completer) {
```

Reply:
During rebase I realize that there was introduced a similar method with `boolean` withing this PR [1] a second arg telling if completer should be added or not... So this change is not actual anymore after that
[1] https://github.com/apache/flink/pull/20182
link to the method https://github.com/apache/flink/pull/20182/files#diff-d5511cf0b3c81a60ae476f8a36c37c723fd703eb2c44c556473cb0066502481aR681